### ISSUE TITLE
Enchantment

### ISSUE THREAD

**Issue by deadfishhackers** [thelinuxchoice/locator#2]
Can u make it so it bypassed [Location Permission] thing

**Comment by pyjthon** [comment]
yup me too i have that problem so whenever i sent the victims the link it takes to a website that ask us for a location permission can any1 help pls? THX ;)
